shared_model/validators/block_validator.hpp

```diff
@@ -41,10 +41,12 @@ namespace shared_model {
        */
       Answer validate(
```

Comment:
Methods which accept `PolymorphicWrapper` should be removed as in https://soramitsu.atlassian.net/browse/IR-872 title - "Remove poly wrapper from shared_model::validator"